### PR TITLE
Update Flexible container size

### PR DESCRIPTION
**Pull request by moshie**
I spoke to Rob and confirmed the container width and gutter size for the new designs

### CODE REVIEW

File: src/constants/grid.ts [hunk 1]
```diff
@@ -13,16 +13,16 @@ const grid: IGrid = {
     l: 992,
     m: 768,
     s: 576,
-    xl: 1200,
+    xl: 1250,
```
Comment (dfvalero) [inline src/constants/grid.ts]: the xl sizes and the gutter size look a bit weird to me :|

Comment (moshie) [reply src/constants/grid.ts]: Might be worth speaking to the designers about this.
This update is based on the most recent designs for shopfront.

Comment (moshie) [reply src/constants/grid.ts]: turns out the gutter spacing is 24 not 22 so I have updated the breakpoint to 1300 which is slightly less of a random number :D

**Comment by lluia** [comment]
@moshie can you change the base branch to `milestone/4.0.0` ? 

I'm thinking this has quite a potential to break layouts so I prefer to have it as part of a major release 👨🏻‍🔧 📐

**Pull request merged**